Prism/Scripts/PrismUtils/PathManager.py

```diff
@@ -155,6 +155,7 @@ def getOutputPath(
         fileType = fileType or "exr"
         singleFileFormats = ["avi", "mp4", "mov"]
         padding = "." if fileType in singleFileFormats else ".####."
+        task = task.replace("/", "\\")
```

Comment:
Instead of `"\\"` you could use `os.path.sep`. This way it will also work under  Linux.